internal/config/config.go

```diff
@@ -64,6 +64,19 @@ type HTTPSink struct {
 	// Retries is the maximum number of attempts to connect to the HTTP sink if the connection fails before giving up
 	// and exiting.
 	Retries int `mapstructure:"retries"`
+	// TLS allows to configure TLS encryption and authentication for the HTTP sink.
+	TLS TLSConfig `mapstructure:"tls"`
+}
+
+type TLSConfig struct {
+	// Enabled dictates whether TLS is used to connect to the HTTP sink.
+	Enabled bool `mapstructure:"enabled"`
+	// CertPath is the path to a PEM-encoded TLS certificate that will be presented to the server.
+	CertPath string `mapstructure:"certPath"`
```

Comment:
could we mount in a similar way of the ETCD mtls?

Not saying that I do not prefer this one, just that I would use the same approach

Reply:
mmmm true, if we can mount it directly it is better, the use case is different therefore we can use a different approach 👍